CSC8503CoreClasses/AnimationSystem.cpp

```diff
@@ -171,17 +176,15 @@ void AnimationSystem::PreloadMatTextures(GameTechRenderer& renderer)
 						std::cout << texID << endl;
 						NCL::Rendering::OGLRenderer::SetTextureRepeating(texID, true);
 					}
-					
+					mAnimTexID.emplace_back(texID);
```

Comment:
I don't know why we need to make new textures every single time we load the level, but hey it works for now we can talk about this later.

Reply:
You're absolutely right, I also think it's a bit stupid to create and remove textures at each level.
The texID is currently attached to the renderObject, and since new levels delete the renderObject the texID will disappear.
I've conceived that the best way to do this would be to have a set of texIDs preloaded at the very beginning of the game and assigned with different vectors, and then point to the appropriate vectors (texIDs) when the game goes to the next level and creates a new Object.